test/verify/check-system-info

```diff
@@ -899,7 +899,7 @@ password=foobar
                 # Initramfs re-generation takes a while
                 m.wait_reboot(timeout_sec=600)
             else:
-                m.wait_reboot()
+                m.wait_reboot(timeout_sec=300)
```

Comment:
So I take it this also needs to re-generate the initrd then? In that case, just get rid of the `if` and just always use the long timeout.

Reply:
Okido, done!